wavefront/server/apps/floconsole/floconsole/controllers/app_user_controller.py

```
@@ -0,0 +1,262 @@
+            ),
+        )
+
+
+@app_user_router.get('/apps/{app_id}/users')
+@inject
+async def list_app_users(
+    app_id: UUID,
+    request: Request,
+    response_formatter: ResponseFormatter = Depends(
+        Provide[CommonContainer.response_formatter]
+    ),
```

Ruff (0.14.10)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)